source/extensions/load_balancing_policies/common/load_balancer_impl.h

```diff
@@ -459,6 +459,7 @@ class ZoneAwareLoadBalancerBase : public LoadBalancerBase {
   const absl::optional<uint32_t> force_local_zone_min_size_;
   // Keep small members (bools and enums) at the end of class, to reduce alignment overhead.
   const uint32_t routing_enabled_;
+  const uint32_t locality_basis_;
```

Comment:
I'm too lazy too look into this, but please make sure that you're using the appropriate type here. I'm not sure what the actual type of the enum would be. If it is uint32, this is obviously fine, but I want us to use the correct type here.